drivers/media/xen/xen_camera_front_v4l2.c

```diff
@@ -805,6 +819,14 @@ int ioctl_enum_frameintervals(struct file *file, void *fh,
 	struct xen_camera_front_cfg_format *format;
 	struct xen_camera_front_cfg_resolution *resolution;
 
+	/*
+	 * If no frame rates are supported (not configured for the given guest)
```

Comment:
Instead of copy-pasting this three times I would better move this explanation to frame_rates_supported comment, saying that frame_rates_supported is only called from IOCTLs and "If no frame rates are..."

Reply:
sounds reasonable, will update